### ISSUE TITLE
Calendar not displaying

### ISSUE THREAD

**Issue by simmonspaul** [Roomify/bat_drupal#12]
[Be aware that I hacked code as per #10 to enable BAT Calendar Reference module - this is in a similar area so might be the underlying cause]

When I get everything setup and having gone to Unit Management and added Units and then finally click on the calendar page, I get the two drop downs but I do not get the fullcalendar.

My drupal status page confirms that FullCalendar plugin v2.6.0 is installed.
I have also included the fullcalendar-scheduler v1.2.0 library as advised.

The reason I am raising this ticket, is that there is no way of knowing if fullcalendar-scheduler is correctly available to BAT or not. 
If I remove the fullcalendar-scheduler library completely, there is no error anywhere.
In an effort to try and get this working, I also chose each of the different fullcalendar license options from the configuration menu... there was no difference.

Thanks again,
Paul

**Comment by simmonspaul** [comment]
This error is obviously a show stopper for me.

I rebuilt my environment and reinstalled bat from a known state, fullcalendar still was not displaying.
(I found that the Field_UI module is a requirement and not in the documentation).

I created a brand new view using fullcalendar in the same environment and it also was not working. 
This rules out a bat code issue.
Hence I will create the environment afresh and ensure fullcalendar is working first to rule out environment issues on my side, then I will install bat to see there is some sort of conflict.

I am using fullcalendar 2.6.0. I will also try 1.6.7.

**Comment by simmonspaul** [comment]
k. Still not happy but I know now that this is an issue with fullcalendar not bat.

I saw from the drupal fullcalendar page that they only had fully tested fullcalendar v1.5.x so I installed 1.5.4 for testing purposes.

I used a fresh environment and installed fullcalendar 1.5.4, I created a new full calendar view and was able to observe a calendar confirming it worked. However when I saved the view, the calendar disappeared. I had to change the view, back to a list view and then back to fullcalendar in order to get the calendar back. The calendar itself showing on the view preview was intermittent.

When I went to the node itself the calendar was not there, however I noticed that firebug was giving me two js errors. $.browser is undefined and $.cookie is not a function. So I am assuming that fullcalendar and firefox have some issues.

I switched to internet explorer and the calendar did show up, however again it was intermittent in the view preview screen.

So not happy, If I cant get a fullcalendar view to show consistently then I am not going to get bat working... will do some more investigation and let you know if I solve my problem.
